### DISCUSSION TITLE
Option to automatically disable barrier when a fullscreen application is detected

### DISCUSSION THREAD

**Discussion by Deliphin** [debauchee/barrier#1596] [Ideas]
Some games properly lock your mouse to the game, so this isn't an issue. But some games, especially older ones, do not do this properly, and so Barrier thinks your mouse is moving to another monitor.

The game I have this problem with is Red Orchestra 2. If I don't disable Barrier, and move my mouse to the left, I will be kicked out of fullscreen, as the active "window" is now barrier, using my input for my other device.

Probably best to only detect for true fullscreen, as most borderless fullscreen games probably handle mouse input correctly.

**Comment by aergndi** [comment]
YES PLEASE SIR
